devservices/config.yml

```diff
@@ -19,9 +19,30 @@ x-sentry-service-config:
         repo_link: https://github.com/getsentry/sentry-shared-kafka.git
     snuba:
       description: Service that provides a rich data model on top of Clickhouse together with a fast ingestion consumer and a query optimizer
+    profiles-consumer:
+      description: Consumer for profiles
```

Comment:
These descriptions are quite redundant. I know we need them but maybe it could be more descriptive of what it is rather than just repeating the name